src/biome/text/pipelines/similarity_classifier.py

```diff
@@ -1,30 +1,18 @@
-from typing import Union, List, Type
+from typing import Union, List
 
-from allennlp.models import Model as AllennlpModel
 from allennlp.predictors import Predictor
 
-from biome.text.pipelines.pipeline import Pipeline
 from biome.text.dataset_readers import SequencePairClassifierReader
-from biome.text.dataset_readers.datasource_reader import DataSourceReader
 from biome.text.models import SimilarityClassifier as TorchSimilarityClassifier
```

Comment:
I think there is no more need for this renaming
```suggestion
from biome.text.models import SimilarityClassifier
```
I think this was done to not clash with the pipeline class name, but now it's `SimilarityClassifierPipeline`